src/pages/Wordtest/WordtestTitle.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import GoToTest from './GoToTest';
+import LearnWords from './LearnWords';
```

Comment:
```suggestion
import LearnWords from './Learnwords';
```
파일명은 `Learnwords.jsx`로 설정되어있는데, `LearnWords.jsx`를 불러오려고 하고 있습니다.
파일명을 변경하거나 위 소스코드 처럼 불러오는 `Learnwords`로 변경해야할 것 같습니다.